src/seedu/addressbook/parser/Parser.java

```diff
@@ -20,10 +20,12 @@
 import seedu.addressbook.commands.HelpCommand;
 import seedu.addressbook.commands.IncorrectCommand;
 import seedu.addressbook.commands.ListCommand;
+import seedu.addressbook.commands.TotalCountCommand;
+import seedu.addressbook.commands.TopTenCommand;
 import seedu.addressbook.commands.ViewAllCommand;
 import seedu.addressbook.commands.ViewCommand;
 import seedu.addressbook.data.exception.IllegalValueException;
-
+import seedu.addressbook.commands.*;
```

Comment:
Some IDEs automatically change imports into wildcard like this, which is against our coding standards. Please configure your IDE so that it stops doing this :)